docs/src/main/asciidoc/spanner.adoc

```diff
@@ -288,14 +316,14 @@ public class ConverterConfiguration {
 
 === Repositories
 
-{spring-data-commons-ref}/#repositories[Spring Data Repositories] are a powerful abstraction that can save you a lot of boilerplate code.
-For Spring Data Spanner, the code inheriting `SpannerRepository` gets all the benefits of `CrudRepository` and `PagingAndSortingRepository` as well.
+{spring-data-commons-ref}/#repositories[Spring Data Repositories] are a powerful abstraction that
+can save you a lot of boilerplate code.
 
 For example:
 
 [source,java]
 ----
-public interface TradeRepository extends SpannerRepository<Trade> {
+public interface TradeRepository extends PagingAndSortingRepository<Trade> {
```

Comment:
I think this is a point I missed from the last PR: 

There needs to be 2 type params here. So SpannerRepository or PagingAndSortingRepository with params <Trade, Key>. If you use SpannerRepository instead of the PandR one then you also get the methods the SpannerRepo has in addition. 

The second type param must be Key (the Spanner Key type), a descendant of Object[ ], or Iterable if there are multiple primary key columns, and can be any descendant of Object if there is only one PK column.

Reply:
So it turns out that the key was required even before.
Passed in the key to `PagingAndSortingRepository`, and the plan is to deprecate `SpannerRepository` (#526).